File: stephanie/Problema0095/Problema0095.js
const prompt = require("prompt-sync")({ sigint: true });

var n = prompt("Digite um número inteiro positivo: ")

switch (n){
    case 0: console.log("Zero");
    break;
    case 1: console.log("Um");
    break;
    case 2: console.log("Dois");
    break;
    case 3: console.log("Três");
    break;
    case 4: console.log("Quatro");
    break;
    case 5: console.log("Cinco");
    break;
    case 6: console.log("Seis");
    break;
    case 7: console.log("Sete");
    break;
    case 8: console.log("Oito");
    break;
    case 9: console.log("Nove");
    break;
    default: console.log("Entrada Inválida!");
    break;
}

File: stephanie/Problema0095/Problema0095.java
package stephanie.Problema0095;

import java.util.Scanner;

public class Problema0095 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Digite um número inteiro positivo: ");
        int n = Integer.valueOf(input.nextLine());
        
        switch (n){
            case 0: System.out.print("Zero");
            break;
            case 1: System.out.print("Um");
            break;
            case 2: System.out.print("Dois");
            break;
            case 3: System.out.print("Três");
            break;
            case 4: System.out.print("Quatro");
            break;
            case 5: System.out.print("Cinco");
            break;
            case 6: System.out.print("Seis");
            break;
            case 7: System.out.print("Sete");
            break;
            case 8: System.out.print("Oito");
            break;
            case 9: System.out.print("Nove");
            break;
            default: System.out.print("Entrada Inválida!");
            break;
        }
        
    }
}
